### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NitinGote**
Signed-off-by: Nitin Gote <nitin.r.gote@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,9 @@ Each project is tagged consistently, so when pulling these repos, pull the same
...
-| Ubuntu® 22.04 (linux-oem image 5.17) | [backport/main](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/backport/main) | [Readme](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/backport/main/docs/README_ubuntu.md)| Yes | Yes |
-| Ubuntu® 22.04/20.04 (5.15 generic)  | [backport/main](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/backport/main) | [Readme](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/backport/main/docs/README_ubuntu.md)| Yes | Yes |
+| Ubuntu® 22.04 (5.19 generic)  | [backport/main](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/backport/main) | [Readme](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/backport/main/docs/README_ubuntu.md)| Yes | Yes |
+| Ubuntu® 22.04 (linux-oem image 5.17) | [backport/main](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/backport/main) | [Readme](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/backport/main/docs/README_ubuntu.md)| Yes | No |
```
Comment (ayazahma) [inline README.md:30]: no longer being tested

Comment (ayazahma) [reply README.md:30]: so lets remove it

Comment (NitinGote) [reply README.md]: done

**Review by smuqthya — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -26,16 +26,10 @@ Each project is tagged consistently, so when pulling these repos, pull the same
 | Red Hat® Enterprise Linux® 8.5       | [redhat/main](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/redhat/main) | [Readme](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/redhat/main/README.md)| Yes | No |
```
[the remark this answers is not in the document]
Comment (ayazahma) [reply README.md:26]: BACKPORTS_RELEASE_TAG="I915_23.3.19_PSB_230122.18"
DII_KERNEL_TAG="I915-23.3.19"
DII_KERNEL_HEAD="44490ef1939be1"
UBUNTU_GENERIC_KERNEL_VERSION="5.15.0-57"
VANILLA_5.15LTS_KERNEL_VERSION="5.15.81"
UBUNTU_OEM_22.04_KERNEL_VERSION="5.17.0-1020"
SLES15_SP4_KERNEL_VERSION="5.14.21-150400.24.33"
VANILLA_KERNEL_VERSION="5.19.0"
UBUNTU_OEM_20.04_KERNEL_VERSION="5.14.0-1055"

[the remark this answers is not in the document]
Comment (ayazahma) [reply README.md:26]: Sorry I missed earlier  but in this release The version file is not reflecting 5.19 ubuntu generic

**Pull request merged**